.vscode/settings.json

```diff
@@ -43,5 +43,6 @@
     "sysgen",
     "utrans",
     "yapf"
-  ]
+  ],
+  "python.pythonPath": "C:\\Users\\gbunce\\AppData\\Local\\ESRI\\conda\\envs\\arcgispro-py3-clone\\python.exe"
```

Comment:
**nitpick**: this doesn't need to be set any longer with vs code. I thought it was out of experimentation so you might just need to update or opt into the experiment with 

```json
"python.experiments.optInto": [
    "DeprecatePythonPath - experiment"
]
```

to your global settings.json (ctrl + shift + p) `Preferences: Open Settings (JSON)`.

A lot of have been doing it without issues so it should be fine.

Reply:
gotcha. thanks for the info. 

so, it's removed from the .vscode/settings.json file and i added that line to my c;\users\....\settings.json.  does that sound right?

Reply:
Yup